hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/performance/OperationCost.java

```diff
@@ -110,6 +110,9 @@
   public static final OperationCost LIST_FILES_LIST_OP =
       new OperationCost(0, 1);
 
+  /** listStatus always does a LIST. */
+  public static final OperationCost LIST_STATUS_LIST_OP =
+          new OperationCost(0, 1);
```

Comment:
is this just LIST_OPERATION? IF so, make the new constant a reference to that one to make clear that's what is happening. Probably good to do the same above too